seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/AbstractJdbcIT.java

```diff
@@ -110,8 +111,12 @@ private void batchInsertData() {
     private void initializeJdbcTable() throws SQLException {
         try (Connection connection = DriverManager.getConnection(jdbcUrl, jdbcCase.getUserName(), jdbcCase.getPassword())) {
             Statement statement = connection.createStatement();
+            String createDatabase = jdbcCase.getInitDatabase();
             String createSource = jdbcCase.getDdlSource();
```

Comment:
Can include directives to create a database

https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-connp-props-security.html#cj-conn-prop_allowMultiQueries

Reply:
> Can include directives to create a database in createSource sql
> 
> https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-connp-props-security.html#cj-conn-prop_allowMultiQueries

 Thank you for your suggestion. I have fixed it